src/sentry/tasks/post_process.py

```diff
@@ -204,6 +204,35 @@ def post_process_group(
 
         set_current_event_project(event.project_id)
 
+        is_transaction_event = not bool(event.group_id)
+
+        # Simplified post processing for transaction events.
+        # This should eventually be completely removed and transactions
+        # will not go through any post processing.
+        if is_transaction_event:
+            from sentry.models import EventDict, Organization, Project
+
+            # Re-bind node data to avoid renormalization. We only want to
+            # renormalize when loading old data from the database.
+            event.data = EventDict(event.data, skip_renormalization=True)
+
+            # Re-bind Project and Org since we're reading the Event object
+            # from cache which may contain stale parent models.
+            event.project = Project.objects.get_from_cache(id=event.project_id)
+            event.project._organization_cache = Organization.objects.get_from_cache(
+                id=event.project.organization_id
+            )
```

Comment:
We could probably move this before the `if is_transaction_event` statement and just share it for events and transactions